ci/nodepool/nodepool.yaml

```diff
@@ -1,12 +1,12 @@
-script-dir: /home/pulp/pulp-ci/ci/nodepool/scripts/
+script-dir: /home/pulp/pulp_packaging/ci/nodepool/scripts/
```

Comment:
Is this a regression?

Reply:
kinda? I would guess that the mass rename accidentally got this one. But we have not been using this file to keep our nodepool up to date so nothing's broken.

Reply:
Hunh. I thought we weren't using a repository named  `pulp_packaging` at all.

Reply:
The repository was renamed from `pulp_packaging` to `pulp_ci`.
The directory on the nodepool server was created when the repo was name `pulp_packaging`.
I am not sure if there's any other things on the nodepool server that are hardcoded to the `/home/pulp/pulp_packaging` location so I have not renamed the directory. But I have it's git remote url to point to `pulp-ci`

Reply:
Ahh, I understand now. Thanks.

Reply:
I asked @pcreech about this and he is 95% sure nothing else is using the pulp_packaging path, so I will rename it on the server.